docker/README.md

```diff
@@ -453,6 +454,7 @@ create a new IdP or modify your existing configuration.
 1. In the `.env` file, set the required environment variables
     - `FIFTYONE_API_URI`
     - `FIFTYONE_AUTH_SECRET`
+    - The `CAS_*` variables referenced [here](#central-authentication-service) in case they were overwritten when copying your `.env` file
```

Comment:
Let's be explicit instead of referential.

```suggestion
    - `CAS_BASE_URL`
    - `CAS_BIND_ADDRESS`
    - `CAS_BIND_PORT`
    - `CAS_DATABASE_NAME`
    - `CAS_DEBUG`
    - `CAS_DEFAULT_USER_ROLE`

    > **Note**: For the `CAS_*` variables, consider using
    > the seed values from the `.env.template` file

```

I don't understand how a customer moving `.env` into the directories `internal-auth` or `legacy-auth` would overwrite.  We don't provide a `.env` file in those directories.  And the `.env.template` remains.

Reply:
Right, forgot it was .env.template so wouldn't overwrite. Changes look good